qa/integration-tests/src/test/java/io/camunda/it/operate/OperateProcessInstanceMigrationIT.java

```diff
@@ -0,0 +1,243 @@
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.operate;
+
+import static io.camunda.zeebe.client.protocol.rest.PermissionTypeEnum.UPDATE;
+import static io.camunda.zeebe.client.protocol.rest.ResourceTypeEnum.BATCH;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import io.camunda.qa.util.cluster.TestRestOperateClient;
+import io.camunda.qa.util.cluster.TestStandaloneCamunda;
+import io.camunda.webapps.schema.entities.operation.BatchOperationEntity;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.command.MigrationPlan;
+import io.camunda.zeebe.client.api.response.ActivateJobsResponse;
+import io.camunda.zeebe.client.api.response.ActivatedJob;
+import io.camunda.zeebe.client.api.response.CompleteJobResponse;
+import io.camunda.zeebe.client.api.response.DeploymentEvent;
+import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import io.camunda.zeebe.client.api.search.response.FlowNodeInstance;
+import io.camunda.zeebe.it.util.AuthorizationsUtil;
+import io.camunda.zeebe.it.util.AuthorizationsUtil.Permissions;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.util.Either;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+public class OperateProcessInstanceMigrationIT {
+
+  private static final String SUPER_USER = "super";
+  private static final String RESTRICTED_USER = "restricted";
+  private static final String PROCESS_DEFINITION_ID_1 = "service_tasks_v1";
+  private static final String PROCESS_DEFINITION_ID_2 = "incident_process_v1";
+
+  @TestZeebe(initMethod = "initTestStandaloneCamunda")
+  private static TestStandaloneCamunda testStandaloneCamunda;
+
+  private static ZeebeClient zeebeClient;
+  private static TestRestOperateClient superOperateClient;
+
+  @SuppressWarnings("unused")
+  static void initTestStandaloneCamunda() {
+    testStandaloneCamunda = new TestStandaloneCamunda();
+    zeebeClient = testStandaloneCamunda.newClientBuilder().build();
+  }
+
+  @BeforeAll
+  public static void before() {
+    final var authorizationsUtil =
+        new AuthorizationsUtil(
+            testStandaloneCamunda, testStandaloneCamunda.getElasticSearchHostAddress());
+    final var defaultClient = authorizationsUtil.getDefaultClient();
+    // create super user that can read all process definitions
+    final var superZeebeClient =
+        authorizationsUtil.createUserAndClient(
+            SUPER_USER, "password", new Permissions(BATCH, UPDATE, List.of("*")));
+    superOperateClient = testStandaloneCamunda.newOperateClient(SUPER_USER, "password");
+  }
```

Comment:
@houssain-barouni I saw that you introduce this recently for another test, I thought I need this as well as the endpoint I want to query is annotated 

```

  @PreAuthorize("hasPermission('write')")
```



but not sure anymore (I don't feel it helps :D ) Can you quickly confirm I'm on the right track? 🙏🏼